napari/_vispy/layers/image.py

```diff
@@ -93,6 +93,18 @@ def __init__(
             layer_node_class=layer_node_class,
         )
 
+        self._array_like = True
+
+        # set the 3D shader step size bounds
+        self.min_step_size = 0.05
+        self.max_step_size = 20
```

Comment:
```suggestion
        self.max_step_size = 80
```
Safe to make this bigger IMO -- ultimately may want to make this a setting (+/- the fps theresholds)